crates/sui-graphql-rpc/src/types/move_type.rs

```diff
@@ -114,6 +117,22 @@ impl MoveType {
 
         MoveTypeLayout::try_from(self.layout_impl(resolver).await.extend()?).extend()
     }
+
+    /// The abilities this concrete type has.
```

Comment:
specifically 'Copy' | 'Drop' | 'Store' | 'Key' right? is it worth mentioning here or should that be assumed to be common knowledge

Reply:
That documentation belongs on the `MoveAbility` type, I think.